[2023-06-29T01:51:52.044+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_rds_data_to_s3 manual__2023-06-28T21:30:21-04:00 [queued]>
[2023-06-29T01:51:52.058+0000] {taskinstance.py:1090} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: get_pnadc_data_new.transfer_rds_data_to_s3 manual__2023-06-28T21:30:21-04:00 [queued]>
[2023-06-29T01:51:52.058+0000] {taskinstance.py:1288} INFO - 
--------------------------------------------------------------------------------
[2023-06-29T01:51:52.058+0000] {taskinstance.py:1289} INFO - Starting attempt 1 of 1
[2023-06-29T01:51:52.058+0000] {taskinstance.py:1290} INFO - 
--------------------------------------------------------------------------------
[2023-06-29T01:51:52.101+0000] {taskinstance.py:1309} INFO - Executing <Task(PythonOperator): transfer_rds_data_to_s3> on 2023-06-29 01:30:21+00:00
[2023-06-29T01:51:52.109+0000] {standard_task_runner.py:55} INFO - Started process 2036 to run task
[2023-06-29T01:51:52.113+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'get_pnadc_data_new', 'transfer_rds_data_to_s3', 'manual__2023-06-28T21:30:21-04:00', '--job-id', '355', '--raw', '--subdir', 'DAGS_FOLDER/get_pnadc_data_new.py', '--cfg-path', '/tmp/tmpa2rhvwpp']
[2023-06-29T01:51:52.115+0000] {standard_task_runner.py:83} INFO - Job 355: Subtask transfer_rds_data_to_s3
[2023-06-29T01:51:52.218+0000] {task_command.py:389} INFO - Running <TaskInstance: get_pnadc_data_new.transfer_rds_data_to_s3 manual__2023-06-28T21:30:21-04:00 [running]> on host 201200ed1de4
[2023-06-29T01:51:52.556+0000] {taskinstance.py:1518} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=get_pnadc_data_new
AIRFLOW_CTX_TASK_ID=transfer_rds_data_to_s3
AIRFLOW_CTX_EXECUTION_DATE=2023-06-29T01:30:21+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-06-28T21:30:21-04:00
[2023-06-29T01:51:52.571+0000] {base.py:73} INFO - Using connection ID 'aws_conn' for task execution.
[2023-06-29T01:51:52.572+0000] {connection_wrapper.py:337} INFO - AWS Connection (conn_id='aws_conn', conn_type='aws') credentials retrieved from login and password.
[2023-06-29T01:51:53.358+0000] {base.py:73} INFO - Using connection ID 'postgres_conn' for task execution.
[2023-06-29T01:52:07.688+0000] {python.py:177} INFO - Done. Returned value was: None
[2023-06-29T01:52:07.707+0000] {taskinstance.py:1332} INFO - Marking task as SUCCESS. dag_id=get_pnadc_data_new, task_id=transfer_rds_data_to_s3, execution_date=20230629T013021, start_date=20230629T015152, end_date=20230629T015207
[2023-06-29T01:52:07.887+0000] {local_task_job.py:212} INFO - Task exited with return code 0
[2023-06-29T01:52:07.911+0000] {taskinstance.py:2596} INFO - 1 downstream tasks scheduled from follow-on schedule check
